tox.ini

```diff
@@ -95,3 +95,29 @@ setenv =
 commands =
     pip install -e .
 	behave --tags=-disabled integration_tests
+
+
+[testenv:ci_gsye_122]
+basepython = python3.8
+passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH BRANCH
+deps = pip-tools
+whitelist_externals = git
+                      /bin/rm
+                      /bin/cp
+                      /bin/mkdir
+                      echo
+					  bash
+commands =
+		python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
+		pip-sync requirements/tests.txt requirements/pandapower.txt
+		pip install -e .
+		pip uninstall -y gsy-framework
+		pip install git+https://github.com/gridsingularity/gsy-framework@{env:BRANCH:master}
+		rm -rf gsy-backend-integration-tests
+		rm -rf integration_tests
+		mkdir integration_tests
+		git clone --branch {env:BRANCH:master} git@github.com:gridsingularity/gsy-backend-integration-tests.git
+		cp -r ./gsy-backend-integration-tests/gsy_e_tests/integration_tests/ ./integration_tests/
+		bash -c 'for f in ./integration_tests/*.feature; do behave --tags=-slow --tags=-disabled "$f" || exit 1; done'
+		rm -rf gsy-backend-integration-tests
+		rm -rf integration_tests
```

Comment:
You probably want this for local runs to clean up right? Lets see how we can get rid of it (see the next comments).

Reply:
Can you please try out providing a temporary directory with `envtmpdir` 
https://tox.wiki/en/latest/config.html#conf-envtmpdirm ? AFAIU you do not need to clean up then. Thanks!

Reply:
I'll give it a try! Thanks for the resource